### ISSUE TITLE
May i know what the meaning of each data split?

### ISSUE THREAD

**Issue by chaneyddtt** [AlexTheBad/AP-10K#4]
Hi,
Thanks for introducing this dataset.  I notice that there are there data splits in the data annotations file. May I know whether they are corresponding to the different experimental settings in the paper? If yes, could you please provide more details on these three different splits?

**Comment by Annbless** [comment]
Hi,

This is a good question. For a comprehensive comparison, we calculate the mean and variance of the estimation accuracy for each pose estimation method (e.g., the results in Tables 3 and 4 in the paper). Therefore, we randomly partition the entire dataset three times to form a training set, a validation set, and a test set, as demonstrated in the description in Section 4.2 of the paper. The three data splits are then used to calculate the mean and variance of the estimation accuracy.
